cartographer_ros/cartographer_ros/map_builder_bridge.cc

```diff
@@ -105,10 +105,9 @@ bool MapBuilderBridge::HandleSubmapQuery(
   }
 
   response.submap_version = response_proto.submap_version();
-  if (response_proto.textures_size() == 0) {
-    LOG(ERROR) << "Empty textures given.";
-    return false;
-  }
+  CHECK(response_proto.textures_size() > 0)
+      << "empty textures given for trajectory_id " << request.trajectory_id
```

Comment:
nit: There is an operator<< for `SubmapId`.
```
<< "Empty textures given for submap: " << submap_id
```
and extract `submap_id` above into a local variable?

Reply:
Done, PTAL.